### DISCUSSION TITLE
Progressive Web App needs refresh button

### DISCUSSION THREAD

**Discussion by oguzhantufenk** [dailydotdev/daily#949] [Feature Requests]
I use the Daily.dev app as a PWA on my device and sometimes articles that don't interest me show up in bulk. It would be nice if there was a small button that would suggest new articles.

**Comment by idoshamun** [comment]
Hi @oguzhannftsiblings,

Thanks a lot for the suggestion. We'll consider it :)

**Comment by oguzhantufenk** [comment]
> Hi @oguzhannftsiblings,
> 
> 
> 
> Thanks a lot for the suggestion. We'll consider it :)

Thank you very much for your attention! 🤝🤩

**Comment by VBoudreau55** [comment]
I think this could be a pull down to refresh the feed like the reddit mobile app
